src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SeparatorWrapCheckTest.java

```diff
@@ -70,7 +70,7 @@ public void testComma() throws Exception {
         final String[] expected = {
             "39:17: " + getCheckMessage(MSG_LINE_PREVIOUS, ","),
         };
-        verify(checkConfig, getPath("InputSeparatorWrap.java"), expected);
+        verify(checkConfig, getPath("InputSeparatorWrapForTestComma.java"), expected);
```

Comment:
InputSeparatorWrapForTestComma does not need to contain examples for DOT. Only for COMMAS as specified in tokens set.